deployment/kustomizations/base/cm.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wren-config
+data:
+  PLATFORM: linux/amd64
+  WREN_ENGINE_PORT: "8080"
+  WREN_ENGINE_SQL_PORT: "7432"
+  WREN_AI_SERVICE_PORT: "5555"
+  #Release version
+  WREN_PRODUCT_VERSION: "0.3.4"
+  WREN_ENGINE_VERSION: "0.4.4"
+  WREN_AI_SERVICE_VERSION: "0.3.7"
+  WREN_UI_VERSION: "0.5.6"
+  WREN_BOOTSTRAP_VERSION: "0.1.4"
+  OPENAI_GENERATION_MODEL: "gpt-3.5-turbo"
+  POSTHOG_HOST: "https://app.posthog.com"
+  TELEMETRY_ENABLED: "false"
+  HOST_PORT: "3000"
+  AI_SERVICE_FORWARD_PORT: "5555"
+  DATA_PATH: "/app/data"
+  WREN_ENGINE_ENDPOINT: "https://wren-ui.myhost.net"
+  DB_TYPE: sqlite
+#  DB_TYPE: postgres
+  SQLITE_FILE: /app/data/db.sqlite3
+  WREN_AI_ENDPOINT: "http://wren-ai-service.ai-system.svc.cluster.local:5555"
```

Comment:
Let's keep following properties, others could be removed.

```suggestion
  # Wren Engine Service Port
  WREN_ENGINE_PORT: "8080"
  # Wren AI Service Port
  WREN_AI_SERVICE_PORT: "5555"

  # service endpoints of AI service & engine service
  WREN_ENGINE_ENDPOINT: "http://wren-engine-svc:8080"
  WREN_AI_ENDPOINT: "http://wren-ai-service:5555"

  # Release version
  WREN_PRODUCT_VERSION: "0.3.4"
  WREN_ENGINE_VERSION: "0.4.4"
  WREN_AI_SERVICE_VERSION: "0.3.7"
  WREN_UI_VERSION: "0.5.6"
  
  # OpenAI
  OPENAI_GENERATION_MODEL: "gpt-3.5-turbo"
  
  # Telemetry
  POSTHOG_HOST: "https://app.posthog.com"
  TELEMETRY_ENABLED: "false"
  
  # `sqlite`, or `postgres` as application database.
  DB_TYPE: sqlite

  # if DB_TYPE = `sqlite`
  SQLITE_FILE: /app/data/db.sqlite3
  DATA_PATH: "/app/data"
  
  # if DB_TYPE = `postgres`
  # PG_URL: "..."
```

Reply:
These variables are used by the wren-ui container if they are used, then they should stay @wwwy3y3 
https://github.com/Canner/WrenAI/blob/main/docker/docker-compose.yaml#L85-L88
```
WREN_PRODUCT_VERSION
WREN_ENGINE_VERSION
WREN_AI_SERVICE_VERSION
WREN_UI_VERSION
WREN_BOOTSTRAP_VERSION
```

Reply:
As for the variables that are not implemented yet. I personally would let them not removed, because without them its a but confusing. With them people would know that this is something that is not implemented and would not bother you asking this over and over again. Because having those is something that people with k8s environment would be looking in my opinion, that's why I added them.

But I would leave this decision to you.